### ISSUE TITLE
Adding learning rate to tensorboard like loss and examples/second

### ISSUE THREAD

**Issue by vkaul11** [tensorflow/tensorboard#5112]
Is it possible to hook learning rate to tensorboard like we have loss and examples/second? We wanted to get that also in the display.

**Comment by arghyaganguly** [comment]
hi @vkaul11 , please check this [stack overflow link](https://stackoverflow.com/questions/49127214/keras-how-to-output-learning-rate-onto-tensorboard) for usage with Keras. Does it suffice your requirement?

**Comment by vkaul11** [comment]
We are using Estimators not Keras. Will it still work ?

**Comment by psybuzz** [comment]
If you are using TF2 or (eager mode in TF1), and the learning rate value that you wish to write to TensorBoard is available outside of your Estimator, you can manually write it using the TF2-style `tf.summary.scalar` https://www.tensorflow.org/api_docs/python/tf/summary/scalar#for_example

If the learning rate is only available inside the Estimator (e.g. the learning rate changes based on step and the value is not known outside of the Estimator), you will need to add a 'summary op' to the graph that will write the learning rate at the time the graph is being executed.  I haven't checked this yet myself, but I think this might be possible if the Estimator has a custom [`model_fn`](https://www.tensorflow.org/guide/migrate#using_a_custom_model_fn) and it calls `tf.compat.v1.summary.scalar()`, in a manner similar to this: https://cs.opensource.google/tensorflow/estimator/+/master:tensorflow_estimator/python/estimator/canned/boosted_trees.py;l=1299;drc=0d7152569c42b51e8ef8ca04d8d0eadc51090438.

**Comment by arghyaganguly** [comment]
@vkaul11 , please confirm if this ticket can be closed based on the previous comment from @psybuzz .Thank you,

**Comment by arghyaganguly** [comment]
@vkaul11 , closing this issue as it has been awaiting your response for sometime without any activity.Please feel free to reopen if required.Thank you.
